<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchBoxWidget</class>
 <widget class="QWidget" name="SearchBoxWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="CustomLabel" name="label">
         <property name="text">
          <string>TextLabel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout" columnstretch="3,1,3,2">
       <item row="0" column="0">
        <widget class="CustomPushButton" name="firstPlusButton">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="CustomPushButton" name="secondValueButton">
         <property name="text">
          <string>60</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="CustomPushButton" name="secondMinusButton">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="CustomPushButton" name="firstValueButton">
         <property name="text">
          <string>0</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>~</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="CustomPushButton" name="firstMinusButton">
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="CustomPushButton" name="secondPlusButton">
         <property name="text">
          <string>+</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="CustomCheckBox" name="checkBox">
         <property name="text">
          <string>1</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="CustomCheckBox" name="checkBox_2">
         <property name="text">
          <string>10</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3">
        <widget class="CustomPushButton" name="searchButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomPushButton</class>
   <extends>QPushButton</extends>
   <header>CustomPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>CustomLabel</class>
   <extends>QLabel</extends>
   <header>CustomLabel.h</header>
  </customwidget>
  <customwidget>
   <class>CustomCheckBox</class>
   <extends>QCheckBox</extends>
   <header>CustomCheckBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
